<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="text_area">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>411</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_preview"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="actionReplace"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionSelect_none"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionPreference"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionToolbar_top"/>
    <addaction name="actionToolbar_left"/>
    <addaction name="actionToolbar_right"/>
    <addaction name="actionToolbar_bottom"/>
    <addaction name="separator"/>
    <addaction name="actionToolbar_moveable"/>
    <addaction name="actionToolbar_floatable"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Insert</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionStrikeout"/>
    <addaction name="separator"/>
    <addaction name="actionColour"/>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionAlign_center"/>
    <addaction name="actionAlign_Justify"/>
    <addaction name="actionAlign_left"/>
    <addaction name="actionAlign_right"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionExport"/>
   <addaction name="actionImport"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSelect_All"/>
   <addaction name="separator"/>
   <addaction name="actionColour"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionAlign_center"/>
   <addaction name="actionAlign_Justify"/>
   <addaction name="actionAlign_left"/>
   <addaction name="actionAlign_right"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionStrikeout"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/new.png</normaloff>:/image/image/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/file.png</normaloff>:/image/image/file.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/folder.png</normaloff>:/image/image/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/save.png</normaloff>:/image/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/save_as.png</normaloff>:/image/image/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/export.png</normaloff>:/image/image/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/import.png</normaloff>:/image/image/import.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/printer.png</normaloff>:/image/image/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/logout.png</normaloff>:/image/image/logout.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/arrow_undo.png</normaloff>:/image/image/arrow_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/arrow_redo.png</normaloff>:/image/image/arrow_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/cut.png</normaloff>:/image/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut </string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/copy.png</normaloff>:/image/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/paste.png</normaloff>:/image/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/select_all.png</normaloff>:/image/image/select_all.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/find.png</normaloff>:/image/image/find.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/clear.png</normaloff>:/image/image/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/preference.png</normaloff>:/image/image/preference.png</iconset>
   </property>
   <property name="text">
    <string>Preference</string>
   </property>
  </action>
  <action name="actionSelect_none">
   <property name="text">
    <string>Select none</string>
   </property>
  </action>
  <action name="actionToolbar_top">
   <property name="text">
    <string>Toolbar top</string>
   </property>
  </action>
  <action name="actionToolbar_left">
   <property name="text">
    <string>Toolbar left</string>
   </property>
  </action>
  <action name="actionToolbar_right">
   <property name="text">
    <string>Toolbar right</string>
   </property>
  </action>
  <action name="actionToolbar_bottom">
   <property name="text">
    <string>Toolbar bottom</string>
   </property>
  </action>
  <action name="actionToolbar_moveable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar moveable</string>
   </property>
  </action>
  <action name="actionToolbar_floatable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toolbar floatable</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_bold.png</normaloff>:/image/image/text_bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_italic.png</normaloff>:/image/image/text_italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_underline.png</normaloff>:/image/image/text_underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionStrikeout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_strikethrough.png</normaloff>:/image/image/text_strikethrough.png</iconset>
   </property>
   <property name="text">
    <string>Strikeout</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/zoom_in.png</normaloff>:/image/image/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/zoom_out.png</normaloff>:/image/image/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionColour">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/colour.png</normaloff>:/image/image/colour.png</iconset>
   </property>
   <property name="text">
    <string>Colour</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/font.png</normaloff>:/image/image/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionAlign_center">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_align_center.png</normaloff>:/image/image/text_align_center.png</iconset>
   </property>
   <property name="text">
    <string>Align center</string>
   </property>
  </action>
  <action name="actionAlign_Justify">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_align_justify.png</normaloff>:/image/image/text_align_justify.png</iconset>
   </property>
   <property name="text">
    <string>Align Justify</string>
   </property>
  </action>
  <action name="actionAlign_left">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_align_left.png</normaloff>:/image/image/text_align_left.png</iconset>
   </property>
   <property name="text">
    <string>Align left</string>
   </property>
  </action>
  <action name="actionAlign_right">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/image/text_align_right.png</normaloff>:/image/image/text_align_right.png</iconset>
   </property>
   <property name="text">
    <string>Align right</string>
   </property>
  </action>
  <action name="actionPrint_preview">
   <property name="text">
    <string>Print preview</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
